doc/whats_new/v0.4.rst

```diff
@@ -19,5 +19,9 @@ Changelog
   via the frontend.
   :pr:`397` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- Bug: fix a bug for which a user could not check is own code in
```

Comment:
```suggestion
- Bug: fix a bug for which a user could not check their own code in
```
'their' is gender neutral option (I know in French general is always masculine)

Reply:
Also I like that you drop the 'h' even when writing! :laughing:

Reply:
:) no need for it